apps/web/src/pages/Project/index.tsx

```diff
@@ -23,30 +24,81 @@ import {
   HStack,
   StyledLink,
   TagsContainer,
+  ProjectOptions,
 } from './styles';
 import LikeButton from './LikeButton/LikeButton';
+import useIsLoggedIn from '@/hooks/useIsLoggedIn';
+
+import { checkValidationForProjectOptions } from './helper';
 
 function Project() {
   const router = useRouter();
   const { query } = useRouter();
 
-  const { data } = useGetProjectQuery({
+  const notifySuccess = () => toast.success('Project deleted successfully');
+  const notifyFailure = () => toast.error('Project deletetion failed');
```

Comment:
if these are only use in one place, just use the toast directly instead on creating variables.